src/components/MultiHopTrade/components/FiatRamps/FiatRampTradeFooter.tsx

```diff
@@ -185,14 +187,8 @@ export const FiatRampTradeFooter = ({
           />
         )}
 
-        {selectedQuote && !isLoading && (
-          <Display.Mobile>
-            <Protocol
-              onClick={onOpenQuoteList || noop}
-              title={selectedQuote.provider}
-              icon={icon}
-            />
-          </Display.Mobile>
+        {selectedQuote && !isLoading && isSmallerThanXl && (
```

Comment:
I think we should always show `<Protocol />`. The UI feels a bit rough having it render only on `selectedQuote && !isLoading`.